docs/cri/config.md

```diff
@@ -16,11 +16,18 @@ it is recommended to use the `systemd` driver on systemd-based hosts for complia
 [the "single-writer" rule](https://systemd.io/CGROUP_DELEGATION/) of cgroups.
 
 To configure containerd to use the `systemd` driver, set the following option in `/etc/containerd/config.toml`:
++ Before containerd 2.0
```

Comment:
Can we add a note (at the top of this doc, maybe? GitHub won't let me comment there) that config for older versions (`version = 2` and version 1) are auto-converted to version 3?  Upgrading with an existing config file should not break users.

Reply:
fixed